__init__.py

```
@@ -38,46 +45,51 @@
-                toponym = "The " + data['ocean']['name']
+                params = {
+                    "username": self.settings["geonames_user"],
+                    "lat": lat,
+                    "lng": lon,
+                    "formatted": True,
+                    "style": "full"
+                }
+                data = requests.get(land_names,
+                                    params=params).json()
+                toponym = data['countryName']
             except:
```

Ruff

Do not use bare `except`

(E722)